presidio-anonymizer/tests/test_batch_anonymizer_engine.py

```diff
@@ -0,0 +1,71 @@
+from presidio_anonymizer import BatchAnonymizerEngine
+from presidio_anonymizer.entities import (
+  OperatorConfig, RecognizerResult, DictRecognizerResult)
+
+
+texts = ['John', 'Jill', 'Jack']

```

Comment:
You do not reach the dict, str and 'else' cases.
Also, add cases for non string values  `if type(text) in (str, bool, int, float):` and the else case for that

Reply:
added cases for 100% coverage